packages/admin-panel/src/widgets/InputField/JsonEditor.jsx

```diff
@@ -38,7 +38,7 @@ const HelperText = styled(Typography)`
   line-height: 1.66;
 `;
 
-export const JsonEditor = ({ inputKey, label, secondaryLabel, value, onChange, stringify }) => {
+export const JsonEditor = ({ inputKey, label, helperText, value, onChange, stringify }) => {
```

Comment:
This fixed issue where helper text wasn't showing up because it was looking for the wrong property